ambry-account/src/main/java/com/github/ambry/account/mysql/AccountDao.java

```diff
@@ -311,28 +311,17 @@ private List<Container> convertContainersResultSet(ResultSet resultSet) throws S
    * Adds/Updates accounts and their containers to the database in batches atomically using transaction.
    * @param accountsInfo information of updated Accounts
    * @param batchSize number of statements to be executed in one batch
-   * @param disableGenerateColumn disable generated column for multi-primary database since it does not supported.
    * @throws SQLException
    */
-  public synchronized void updateAccounts(List<AccountUpdateInfo> accountsInfo, int batchSize,
-      boolean disableGenerateColumn) throws SQLException {
+  public synchronized void updateAccounts(List<AccountUpdateInfo> accountsInfo, int batchSize) throws SQLException {
     try {
       long startTimeMs = System.currentTimeMillis();
 
       AccountUpdateBatch accountUpdateBatch;
-      if (disableGenerateColumn) {
-        accountUpdateBatch =
-            new AccountUpdateBatch(dataAccessor.getPreparedStatement(insertAccountsSqlNew, true),
-                dataAccessor.getPreparedStatement(updateAccountsSqlNew, true),
-                dataAccessor.getPreparedStatement(insertContainersSqlNew, true),
-                dataAccessor.getPreparedStatement(updateContainersSqlNew, true));
-      } else {
-        accountUpdateBatch =
-            new AccountUpdateBatch(dataAccessor.getPreparedStatement(insertAccountsSql, true),
-                dataAccessor.getPreparedStatement(updateAccountsSql, true),
-                dataAccessor.getPreparedStatement(insertContainersSql, true),
-                dataAccessor.getPreparedStatement(updateContainersSql, true));
-      }
+      accountUpdateBatch = new AccountUpdateBatch(dataAccessor.getPreparedStatement(insertAccountsSqlNew, true),
```

Comment:
Minor: May be we can remove old sql statements like `insertAccountsSql`, `updateAccountsSql`, `insertContainersSql`, `updateContainersSql` or rename the new ones to these variable names?

Reply:
Sure, thx for catching that, updated.